backend/game_engine/common.py

```diff
@@ -211,5 +224,64 @@ def to_json(self):
         return json.dumps(self.to_dict())
 
 
-def log_message(message):
-    logging.info(message)
+def log_message(message, game_attributes):
```

Comment:
Let's make log_message stateless and explicitly define the mandatory and optional arguments to the function. For example:

log_message( message: Text, game_id: Text = None, additional_tags: Dict = None, push_to_sentry=False)

In this case message is a mandatory argument. Game ID may be optionally supplied. Additional properties is a user defined dict of any metadata that should be logged.